src/views/Accessibility/TestingTemplates/index.tsx

```diff
@@ -9,12 +9,13 @@ import {
   SummaryBox
 } from '@trussworks/react-uswds';
 
-import RemediationPlanDoc from 'assets/files/CMS508RemediationPlanTemplate.pdf';
-import TestPlanDoc from 'assets/files/Section508TestPlanTemplate.pdf';
 import PageHeading from 'components/PageHeading';
 
 import './index.scss';
 
+const RemediationPlanDoc = require('assets/files/CMS508RemediationPlanTemplate.pdf');
```

Comment:
In `src/global.d.ts`, if you move the `declare module '*.pdf';` line inside the `declare global {}` block, that'll avoid having to change these `import`s to `require()`s.

Reply:
Ahh! I remember scratching my head on why it was still throwing an error.  So close and obvious haha.